### PR TITLE
Add zoomus-outlook-plugin latest

### PR DESCRIPTION
**Pull request by seanorama**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] [If the `sha256` changed but the `version` didn’t][version-checksum],  
      provide public confirmation by the developer: {{link}}

Additionally, if **adding a new cask**:

- [ ] Named the cask according to the [token reference].
  - changed to have 'zoomus' prefix instead of 'zoom' to be inline with the related 'zoomus' cask.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/zoomus-outlook-plugin.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  auto_updates true
+
+  pkg 'ZoomMacOutlookPlugin.pkg'
+
+  uninstall script:    {
+                         executable: '/Applications/ZoomOutlookPlugin/Uninstall.app/Contents/MacOS/Uninstall',
+                         sudo:       true,
+                       },
+            pkgutil:   'ZoomMacOutlookPlugin.pkg',
+            delete:    '/Applications/ZoomOutlookPlugin/',
```
Comment (commitay) [inline Casks/zoomus-outlook-plugin.rb]: @seanorama The contents of this folder (two `app`s) and the folder itself are removed by the `uninstall script`, is this needed?

**Comment by commitay** [comment]
The installed `app` is `PluginLauncher.app` so I think the current token is 👍

File: Casks/zoomus-outlook-plugin.rb (hunk 1 above)
Comment (seanorama) [inline Casks/zoomus-outlook-plugin.rb]: 1st time dealing with a cask that has an uninstall script so went over the top.
So just drop the delete?

Comment (commitay) [inline Casks/zoomus-outlook-plugin.rb]: Yes, `delete` can be removed.

**Pull request merged**